app/schema/answer.py

```diff
@@ -58,14 +56,12 @@ def check_user_input(user_input):
 
     def get_typed_value(self, raw_input):
 
-        user_input = bleach.clean(raw_input)
```

Comment:
Really it should be bleached immediately after being received from the POST before anything else happens; you don't know what effect dirty data can have on any of the system, even validation code etc.